jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -229,6 +230,15 @@ def _is_unique(m):
                 if _is_unique(new_shadow):
                     new_marks.append(new_shadow)
 
+            # change opacity for live-collapsed spectra from spatial subsets in Cubeviz:
+            spatial_layers = [sub_layer for sub_layer in
+                              self._get_spatial_subset_layers(layer_state.layer.data.label)]
+            spatial_marks = self._get_marks_for_layers(spatial_layers)
+            for layer, mark in zip(spatial_layers, spatial_marks):
+                # update profile opacities for spatial subset:
+                if isinstance(mark, Lines):
+                    mark.set_trait('opacities', [0.8])
```

Comment:
I'm a bit torn on this.  If we change this to edit the layer state instead, then plot options will pick up the value and make more sense, but it does tie the line opacity with the overlay in the image viewer (and therefore this PR would increase the opacity of the spatial subsets to 0.8 as well, which might not be ideal).

What you have here is nice because it doesn't affect the overlay, but then when you go in plot options its a little confusing because it shows opacity of 0.5, and "increasing" that to 0.51, for example, ends up decreasing the opacity (since it was actually 0.8 previously), and from then on the line and overlay are synced.

```suggestion
                if isinstance(mark, Lines):
                    layer.alpha = 0.8
```

Reply:
I think this is a problem with the opacity slider, not with this PR. Given that the opacity slider is used in the context of having the spectrum viewer selected in plot options, it should really only update the opacity for the subset marks in the spectrum viewer, not the subset marks in all viewers for that subset. IMO anyway. It might need an upstream refactor to update viewer-specific marks rather than sharing the opacities between viewers for the subset layers.